spec/fixtures/https_server.lua

```diff
@@ -19,10 +19,13 @@ local cjson = require "cjson"
 math.randomseed(os.time())
 
 
+local MAX_RETRY = 10
```

Comment:
Could we move `MAX_RETRY` near the loop? it may be easy to understand.

Reply:
Also rename it to `HTTPS_SERVER_START_MAX_RETRY` to make the name more specific.